shell/platform/windows/flutter_window_win32_unittests.cc

```diff
@@ -327,5 +334,15 @@ TEST(FlutterWindowWin32Test, OnPointerStarSendsDeviceType) {
                              kDefaultPointerDeviceId);
 }
 
+TEST(FlutterWindowWin32Test, OnWindowRepaint) {
+  MockFlutterWindowWin32 win32window;
+  MockWindowBindingHandlerDelegate delegate;
+  win32window.SetView(&delegate);
+
+  EXPECT_CALL(delegate, OnWindowRepaint()).Times(1);
+
+  win32window.InjectWindowMessage(WM_PAINT, 0, 0);
+}
```

Comment:
The inject window message is really nice! 🏅

Could you also add a test that calls `OnPaint` directly too since it's a public API?

Reply:
done